get_dep.py

```diff
@@ -0,0 +1,8 @@
+import subprocess
+
+def get_dependecies() -> list:
+    # The subprocess.Popen is used for backward compatibility with python 2.7
+    freeze = subprocess.Popen("pip freeze", shell=False, stdout=subprocess.PIPE, stderr=subprocess.PIPE).communicate()
```

Comment:
`pip freeze` probably will not work if you download this project as a package from pypi.
You can test it running `pip install .` in an empty virtual environment.
Instead you could read the requirements.txt or just passing a list of explicit dependencies as shown in the ticket #105